sql-analyzer/pom.xml

```
@@ -0,0 +1,60 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>youngjinj</groupId>
+	<artifactId>sql-analyzer</artifactId>
+	<version>2022.09.15</version>
```

To be official
```suggestion
	<groupId>org.cubrid</groupId>
	<artifactId>sql-analyzer</artifactId>
	<version>1-SNAPSHOT</version>
	<packaging>pom</packaging>
	<name>SQL Analyzer</name>
```

Please add license info and developer lists if you would.
```
<licenses>
    <license>
      <name>The Apache Software License, Version 2.0</name>
      <url>http://www.apache.org/licenses/[LICENSE-2](http://jira.cubrid.org/browse/LICENSE-2).0.txt</url>
      <distribution>repo</distribution>
    </license>
  </licenses>
```

```
  <developers>
    <developer>
      <id>youngjinj</id>
      <name>Youngjin Joo</name>
      <email>...</email>
      <organization>CUBRID</organization>
    </developer>
</developers>
```

---

Thank you. I will change it.